query-connector/src/app/queryBuilding/components/ConditionSelection.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
+
+import styles from "../buildFromTemplates/buildfromTemplate.module.scss";
+import { Dispatch, SetStateAction, useEffect, useRef, useState } from "react";
+import classNames from "classnames";
+import { getConditionsData } from "@/app/database-service";
+import {
+  CategoryNameToConditionOptionMap,
+  mapFetchedDataToFrontendStructure,
+  ConditionIdToValueSetArray,
+} from "../utils";
+import ConditionColumnDisplay from "../buildFromTemplates/ConditionColumnDisplay";
+import SearchField from "@/app/query/designSystem/searchField/SearchField";
+import { BuildStep } from "@/app/constants";
+import { FormError } from "../buildFromTemplates/page";
+
+type ConditionSelectionProps = {
+  fetchedConditions: CategoryNameToConditionOptionMap;
+  selectedConditions: CategoryNameToConditionOptionMap;
+  setBuildStep: (buildStep: BuildStep) => void;
+
+  setFetchedConditions: Dispatch<
+    SetStateAction<CategoryNameToConditionOptionMap | undefined>
+  >;
+  setSelectedConditions: Dispatch<
+    SetStateAction<CategoryNameToConditionOptionMap | undefined>
+  >;
+  queryName: string;
+  validateForm: () => void;
+  setFormError: Dispatch<SetStateAction<FormError>>;
+  formError: FormError;
+  setLoading: Dispatch<SetStateAction<boolean>>;
+  loading: boolean;
+  setConditionValueSets: Dispatch<
+    SetStateAction<ConditionIdToValueSetArray | undefined>
+  >;
+  updateFetched: (selectedConditions: CategoryNameToConditionOptionMap) => void;
+};
+
+/**
+ * Display component for a condition on the query building page
+ * @param root0 - params
+ * @param root0.fetchedConditions - ID of the condition to reference
+ * @param root0.selectedConditions - name of condition to display
+ * @param root0.setFetchedConditions - listener function for checkbox
+ * selection
+ * @param root0.setSelectedConditions - current checkbox selection status
+ * @param root0.queryName - current checkbox selection status
+ * @param root0.formError - indicates missing or incorrect form data
+ * @param root0.setFormError - state function that updates the status of the
+ * condition selection form input data
+ * @param root0.updateFetched - tk
+ * @returns A component for display to redner on the query building page
+ */
+export const ConditionSelection: React.FC<ConditionSelectionProps> = ({
```

Comment:
this used to live on the `buildFromTemplatesPage`; only new things here are the `selectedCondtions` and `formError` state/actions